examples/qml_features/rust/src/custom_base_class.rs

```diff
@@ -269,6 +285,15 @@ impl Default for qobject::State {
 
 /// A struct which inherits from QAbstractListModel
 #[derive(Default)]
+pub struct AbstractBaseClassRust {}
+
+impl qobject::AbstractBaseClass {
+    /// Virtual method for logging
+    pub fn log(self: &AbstractBaseClass) {}
+}
```

Comment:
curious, as it's pure virtual in C++ we don't actually need/want an implementation.

I guess this needed to be here as we generate one of those magic Rust bridges where the implementation is on the Rust side by using extern "Rust" ?

Instead there should be no implementation in Rust, and probably just an extern "C++" block in the generated Rust.

Ah yes i can see that in the test_outputs...